src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -342,12 +363,22 @@ ucs_status_t uct_rc_verbs_ep_am_zcopy(uct_ep_h tl_ep, uint8_t id, const void *he
                                       desc, id, header, header_length, comp, &send_flags);
 
     sge[0].length = sizeof(uct_rc_hdr_t) + header_length;
-    sge_cnt = uct_ib_verbs_sge_fill_iov(sge + 1, iov, iovcnt);
+
+    status = uct_ib_verbs_sge_fill_iov(tl_ep->iface, (void *)&desc_iov,
+                                       sge + 1, iov, iovcnt, &sge_cnt,
+                                       uct_rc_iface_verbs_iov_callback);
+    if (UCS_OK != status) {
+        return status;
+    }
     UCT_RC_VERBS_FILL_AM_ZCOPY_WR_IOV(wr, sge, (sge_cnt + 1), wr.opcode);
     UCT_TL_EP_STAT_OP(&ep->super.super, AM, ZCOPY,
                       (header_length + uct_iov_total_length(iov, iovcnt)));
 
     uct_rc_verbs_ep_post_send_desc(ep, &wr, desc, send_flags);
+    if (NULL != desc_iov) {
+        uct_rc_txqp_add_send_op_sn(&ep->super.txqp, &desc_iov->super, ep->txcnt.pi);
```

Comment:
if all elements in the IOV were LAZY, need to return UCS_OK and not ask for completion